packages/account/src/Components/address-details/address-details.tsx

```diff
@@ -131,7 +122,7 @@ const AddressDetails = ({
         return selected_step_ref?.current?.isSubmitting || (errors && Object.keys(errors).length > 0);
     };
 
-    const checkSubmitStatus = (errors?: { [key: string]: string }) => {
+    const checkSubmitStatus = (errors?: { [key: string]: string } | FormikValues) => {
```

Comment:
```suggestion
    const checkSubmitStatus = (errors?: FormikValues) => {
```
@yauheni-deriv,  last I checked, `FormikValues` had a definition of 
`export interface FormikValues {
    [field: string]: any;
}`
Do we need to add the `{ [key: string]: string }` here? 🤔

Reply:
@jim-deriv, yes, FormikValues are useless. we are going to refactor all our forms in account and get rid of FormikValues. We will set exact types for fields.
it will be implemented separately in further PRs